_drafts/2023-05-23-virtual-co-writing.md

```diff
@@ -0,0 +1,30 @@
+---
+author: Tim Schilling
+category: General
+date: 2023-05-23 12:00:00
+image: /static/img/blog/djangocon_us_earlybird_pricing_2023.png
+layout: post
+post_photo_alt: Organizers at DjangoCon US 2022 in San Diego preparing before the conference. But it looks like they could be co-writing!
+post_photo_url: /static/img/blog/djangocon_us_organizers_2022.jpg
+title: DjangoCon US virtual co-writing
+---
+
+Do you want to write more? Do you want an introduction to other Djangonauts? Then perhaps you should attend the virtual co-writing sessions at DjangoCon US this year!
+
+The purpose of these sessions is to help folks write and be introduced to others in the community. These sessions draw inspiration from [Sumana Harihareswara and Jacob Kaplan-Moss' writing accountability team](https://jacobian.org/2021/mar/9/coworking-to-write-more/). These sessions will be focused on helping people write by providing some light structure.
+
+Here's how our sessions will operate:
+
+- Days: Tuesday, Wednesday, Thursday (Oct 17-19)
+- Start time: [7:30am (US/Eastern)](https://time.is/0730_17_Oct_2023_in_Durham)
```

Comment:
the pedant in me wants to note that `US/Eastern` is a deprecated IANA time zone :D

Reply:
Womp womp. I hate using CST and CDT since I'm lazy. Would US/New York work?